pentaho/src/test/java/pt/webdetails/cpf/olap/AbstractOlapUtilsTest.java

```diff
@@ -102,15 +102,10 @@ public void testGetCubeStructure() throws Exception {
         "\"type\":\"measure\"},{\"qualifiedName\":\"[Unknown]\",\"name\":\"unknown\",\"memberType\":\"UNKNOWN\"," +
         "\"caption\":\"member with type unknown\",\"type\":\"measure\"}]}";
 
-    try {
-      result =
-          olapUtils.getCubeStructure( "", "", "" );
-    } catch ( JSONException e ) {
-      Assert.fail( e.getMessage() );
-    }
-    Assert.assertTrue( jsonEquals( expectedResult, result.toString() ) );
-  }
+    result = olapUtils.getCubeStructure( "", "", "" );
 
+    assertTrue( jsonEquals( expectedResult, result.toString() ) );
```

Comment:
Can you use here the assertEquals?

Reply:
No, because the comparison involves some intermediate transformations.